ibus.py

```diff
@@ -123,16 +123,16 @@ def do_process_key_event(self, keyval, keycode, state):
                     self.commit_candidate()
                     return True
                 return False
-            elif keyval == IBus.Page_Up or keyval == IBus.KP_Page_Up or keyval == IBus.Left:
+             elif keyval == IBus.Page_Up or keyval == IBus.KP_Page_Up or keyval == IBus.Left or keyval == IBus.KP_Left:
```

Comment:
Now you added a space before `elif`. Are you sure this even runs? Please run it and see if the uniemoji appears.

Reply:
huh? i did i misunderstand? and yeah i'm pretty sure it ran the last time i test 🤔

Reply:
weird i would've sworn it ran before i pushed 👀

Reply:
anyway fixed, sorry for the trouble @salty-horse